common/config/rush/browser-approved-packages.json

```diff
@@ -370,6 +378,10 @@
       "name": "cypress",
       "allowedCategories": [ "production" ]
     },
+    {
+      "name": "d3-color",
+      "allowedCategories": [ "production" ]
+    },

```

Comment:
is this package being used anywhere?

Reply:
i was playing around with pnpm audit and this package was added as part of pnpm audit --fix, but this package is not being used anywhere

Reply:
can we remove it then?